src/timeout.c

```diff
@@ -54,18 +55,24 @@ int checkBlockedClientTimeout(client *c, mstime_t now) {
  * each iteration would be costly without any actual gain. */
 int clientsCronHandleTimeout(client *c, mstime_t now_ms) {
     time_t now = now_ms/1000;
-
+    int drop_client = 0;
     if (server.maxidletime &&
         /* This handles the idle clients connection timeout if set. */
         !(c->flags & CLIENT_SLAVE) &&   /* No timeout for slaves and monitors */
         !mustObeyClient(c) &&         /* No timeout for masters and AOF */
         !(c->flags & CLIENT_BLOCKED) && /* No timeout for BLPOP */
         !(c->flags & CLIENT_PUBSUB) &&  /* No timeout for Pub/Sub clients */
-        (now - c->lastinteraction > server.maxidletime))
+        (now - c->lastinteraction > server.maxidletime)) 
```

Comment:
let's remove the extra space (diff)
```suggestion
        (now - c->lastinteraction > server.maxidletime))
```